functions/system.sh

```diff
@@ -162,6 +162,9 @@ permissions_corrections() {
     echo "FAILED (please execute after openHAB was installed)"
     exit 1
   fi
+
+  if is_pine64; then pine64_gpio_permissions;fi
```

Comment:
Would it be easier to just move the four lines here?

Reply:
It dosen't matter for me I can move them. I thought if we add more specific boards later it's easy to get an overview by having functions instead of inline code.